db/migrate/20200823015502_create_house_participations.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# Migration to create house participations
+class CreateHouseParticipations < ActiveRecord::Migration[6.0]
+  def change
+    create_table :house_participations do |t|
+      t.integer :user_id, null: false
+      t.integer :house_id, null: false
+      t.date :join_date, null: false
+      t.integer :score, null: false
```

Comment:
This has me thinking, we do a lot of things with regards to submissions being converted to points at the get go.
But it's entirely possible for people to submit multiple short submissions.
Let's say they spent 20 minutes on three submissions. Each of these would contribute 0 points, but he's done an hour of work and should be entitled to an hour!
Could we either opt for fractional amounts or have the system track these pieces properly? I don't want to force people to compile things together for the sake of houses alone.

Reply:
I asked las houses about this and they were pretty set on 30mins = 1pt. No fractions, no nothing. The way it currently works in las is people are allowed to round up/down if its a few minutes under/over. Combining submissions is not uncommon in houses. 

I suppose I could add up their total amount of time for the month and then integer divide by 30.

I would like keep our implementation as close to las houses as possible. I can ask them again what they think.

Reply:
We're using raw time. No point conversion.